war/src/main/js/util/jenkins.js

```diff
@@ -184,12 +184,36 @@ exports.loadTranslations = function(bundleName, handler, onError) {
 			}
 			throw 'Unable to load localization data: ' + res.message;
 		}
+		
+		var translations = res.data;
+		
+		if('undefined' !== typeof(Proxy)) {
+			translations = new Proxy(translations, {
+				get: function(target, property, receiver) {
+					if(property in target) {
+						return target[property];
+					}
+					console.log('"' + property + '" not found in translation bundle.');
+					return property;
+				}
+			});
+		}
 
-		handler(res.data);
+		handler(translations);
 	});
 };
 
 /**
+ * Load a handlebars template from the server
+ */
+exports.loadTemplate = function(templateName, handler, onError) {
```

Comment:
Yeah, agree.. this is actually not being used anyway, it was a byproduct of my first idea to make this happen... I'll remove it.